TWLight/applications/templates/applications/apply.html

```diff
@@ -20,4 +21,11 @@
       </div>
     </div>
   </div>
+  {% if authorization_method == EMAIL %}
```

Comment:
The variable `authorization_method` is always going to be blank because it doesn't exist in the `context`. Change this to `partner.authorization_method`.